nibe/connection/nibegw.py

```diff
@@ -77,6 +82,7 @@ async def start(self):
         )
 
     def connection_made(self, transport):
+        self.notify_event_listeners(self.CONNECTION_STATUS_EVENT, status="connected")
```

Comment:
I think this callback is called as soon as the udp socket starts listening. So we dont know we have received anything. I think we could notify on product data too?

Reply:
Only if you request product data =)